### DISCUSSION TITLE
Rating - what am I missing for it not to work?

### DISCUSSION THREAD

**Discussion by Mashiane** [saadeghi/daisyui#1140] [Q&A]
Hi

For rating, this is my output.

![image](https://user-images.githubusercontent.com/5799306/190360780-6c203035-c235-49c3-b278-4f8196122eee.png)

This is my html code

![image](https://user-images.githubusercontent.com/5799306/190360958-8a29e04f-85c0-46d8-99ce-c61bbc4e7269.png)

What am I missing? The orange color for the first items and the last ones are not working.

Im using the CDN version.

**Comment by saadeghi** [comment]
The code is correct.  
It works here as you can see: https://codepen.io/saadeghi/pen/abGLdYo?editors=1000

does `bg-orange-400` and `bg-lime-400` work on other elements? because it looks like these colors are not working at all.
